### DISCUSSION TITLE
Getting the need to "reduce initial server response time"

### DISCUSSION THREAD

**Discussion by JamesSingleton** [vercel/next.js#58564] [Help]
### Summary

I ran a couple of PageSpeed Insights on two of my most visited pages on my website:
* [Category Page](https://pagespeed.web.dev/analysis/https-www-redshirtsports-xyz-news-fcs/ommwv22v7s?form_factor=mobile)
* [Home Page](https://pagespeed.web.dev/analysis/https-www-redshirtsports-xyz/osm4yaiaau?form_factor=mobile)

Lately I have been getting a lot of slow page loads saying that the server (Vercel) is taking 4 seconds to respond

<img width="972" alt="Screenshot 2023-11-16 at 8 08 24 PM" src="https://github.com/vercel/next.js/assets/21000200/91c191b4-11d3-44ae-9cef-d0651ba3dbd7">

I switched over entirely to the app router around September and ever since I have gotten these warnings from Google and Ahrefs saying I have slow page speeds and that the server is taking forever to respond. My website is backed by Sanity but I don't think that is what's causing the slowness (maybe 🤷🏼‍♂️ ).

I am at a loss at the moment as to what could be causing this... When I build locally I get this warning

```
 ⚠ Using edge runtime on a page currently disables static generation for that page
```
But the only "page" I use `export const runtime = 'edge'` on is my OpenGraph image route. I disabled doing `generateStaticParams()` for my articles because it was causing my builds on Vercel to take about 25+ minutes to build 300ish pages.

This is the output from building

```
Route (app)                              Size     First Load JS
┌ ○ /                                    1.95 kB         205 kB
├ ○ /_not-found                          879 B           137 kB
├ λ /[slug]                              3.14 kB         206 kB
├ ○ /about                               282 B           246 kB
├ ℇ /api/og                              0 B                0 B
├ λ /api/revalidate                      0 B                0 B
├ ○ /apple-icon1.png                     0 B                0 B
├ λ /authors/[slug]                      2.25 kB         205 kB
├ ○ /contact                             281 B           246 kB
├ ○ /feeds/feed.xml                      0 B                0 B
├ ○ /manifest.webmanifest                0 B                0 B
├ λ /news                                281 B           246 kB
├ λ /news/[division]                     281 B           246 kB
├ λ /news/[division]/[conference]        281 B           246 kB
├ ○ /privacy                             281 B           246 kB
├ ○ /robots.txt                          0 B                0 B
├ λ /search                              1.29 kB         247 kB
├ ○ /sitemap.xml                         0 B                0 B
└ ○ /studio/[[...index]]                 799 kB         1.09 MB
```

**Comment by antoinekm** [comment]
Your links don't work anymore, but I totally agree with that.
